lambda/postSchedulePeerTeachers/Scheduler/src/main/java/db/Person.java

```diff
@@ -115,18 +118,17 @@ public float getAvailabilityScore() {
     }
 
     public int getCurrentAssignments() {
-        return currentAssignments;
+        return sections.size();
     }
     
-    public void addCurrentAssignment() {
-        currentAssignments++;
+    public void addCurrentAssignment(Section section) {
+        sections.put(section.getSectionId(), section);
     }
```

Comment:
Gonna harp on naming again- is sections a list of sections or specific assignments?